crypto/md5/asm/md5-x86_64.pl

```diff
@@ -392,4 +395,5 @@ sub round4_step
 
 print $code;
 
+&file_end($flavour);
```

Comment:
can't you do just write ".file_end" here and do the replacement in x86_64-xlate.pl ?

Reply:
Currently everything is done in x86_64asm.pl.  Doing it in x86_64-xlate.pl needs to change
every invocation of x86_64-xlate.pl to pass CET info to it.   I don't see any advantage.

Reply:
Except this mixes together two styles of generating assembler. I'm not so sure it's a good thing, even if it's an easy hack now...